static/app/utils/profiling/profile/profile.tsx

```diff
@@ -58,7 +61,7 @@ export class Profile {
     name: string;
     startedAt: number;
     threadId: number;
-    type: string;
+    type: 'flamechart' | 'flamegraph' | '';
```

Comment:
What is `''` supposed to represent?

Reply:
Renaming this to 'empty'. We use the static empty instance and I didnt want to pick either as none really applies